packages/app/cypress/e2e/studio/studio.cy.ts

```diff
@@ -737,6 +737,27 @@ describe('studio functionality', () => {
     cy.location().its('hash').and('not.contain', 'testId=').and('not.contain', 'studio=')
   })
 
+  it('does not prompt for URL when creating a new test in studio', () => {
```

Comment:
```suggestion
  it('does not prompt for URL when creating a new test in studio that already visits', () => {
```

Reply:
I think I need to re-word this a different way. The issue was that when the user opens the studio panel to "new test" mode, we were already prompting for a URL in the address bar, even when the spec file was currently running and the AUT was at a URL already.

We only want to prompt for a URL once the test is actually created and it doesn't visit anything yet